src/test/java/com/mongodb/hibernate/jdbc/MongoConnectionTests.java

```diff
@@ -153,7 +154,7 @@ private void checkMethodsWithOpenPrecondition() {
 
     private void assertThrowsClosedException(Executable executable) {
         var exception = assertThrows(SQLException.class, executable);
-        assertEquals("MongoConnection has been closed", exception.getMessage());
+        assertThat(exception.getMessage()).isEqualTo("MongoConnection has been closed");
```

Comment:
Let's use either `assertEquals` or `assertThat(...).isEqualTo` consistently.

`assertEquals` seems much simpler to follow, so I would prefer this.

Reply:
updated. Originally I did this to align with similar changes by my colleague (he used assertJ but using `assertThat(...).matches(...)`. I am ok with either.